vault_program/src/initialize_vault.rs

```diff
@@ -25,7 +25,7 @@ pub fn process_initialize_vault(
     reward_fee_bps: u16,
     decimals: u8,
 ) -> ProgramResult {
-    let [config, vault, vrt_mint, mint, admin, base, system_program, token_program] = accounts
+    let [config, vault, vrt_mint, mint, admin, base, token_program, system_program] = accounts
```

Comment:
wdyt on allowing for diff token programs for the supported mint and the VRT? for instance:

supported_mint: token
vrt_mint: token22

supported_mint: token22
vrt_mint: token

right now this will match both